docs/source/getting-started/installing.adoc

```diff
@@ -16,8 +16,8 @@ The following section describes how to install Minishift and the required depend
 == Prerequisites
 
 Minishift requires a hypervisor to start the virtual machine on which the OpenShift cluster
-is provisioned. Make sure that the hypervisor of your choice is installed and enabled on
-your system before you start Minishift.
+is provisioned. Ensure you have Virtualization Technology (VT-x) enabled in your system and
+the hypervisor of your choice is installed and working before you start Minishift.
```

Comment:
I think we do not need to document this as users are supposed to have a working hypervisor setup. For that they should refer respective hypervisor documentations. I mean we have to draw a line some where about what we want to document and what we do not want to.